### ISSUE TITLE
addresses should default to tcp

### ISSUE THREAD

**Issue by ebuchman** [tendermint/tendermint#551]

**Comment by ethanfrey** [comment]
This is fixed by #552, right?  Or do we only close issues when they merge into master?

If so, we should have a flag "fixed on develop"

**Comment by adrianbrink** [comment]
I think once it is merged into develop we should close the issues.
